.github/workflows/test-workflow.yml

```diff
@@ -8,10 +8,10 @@ on:
         type: string
 
 env:
-  RELAY_IMAGE_TAG: 0.23.3
-  NETWORK_NODE_IMAGE_TAG: 0.39.0-alpha.0
-  HAVEGED_IMAGE_TAG: 0.39.0-alpha.0
-  MIRROR_IMAGE_TAG: 0.81.0-rc1
+  RELAY_IMAGE_TAG: 0.29.1
```

Comment:
```suggestion
  RELAY_IMAGE_TAG: 0.31.1
```

Reply:
Done.

Reply:
Doesn't look like the change was pushed.

Reply:
Yes, the test-workflow.yml file was not staged. It's been committed and pushed. Apologies.